projects/canopy-test-app/src/app/app.component.html

```diff
@@ -287,6 +287,22 @@
               <lg-radio-button value="blue">Blue</lg-radio-button>
             </lg-radio-group>
 
+            <lg-filter-group formControlName="filter">
+              Test filter group select one
+              <lg-filter-button value="red">Red</lg-filter-button>
+              <lg-filter-button value="yellow">Yellow</lg-filter-button>
+              <lg-filter-button value="green">Green</lg-filter-button>
+              <lg-filter-button value="blue">Blue</lg-filter-button>
+            </lg-filter-group>
+
+            <lg-filter-group variant="select-multiple">
+              Test filter group select multiple
+              <lg-filter-button formControlName="redFilter" value="true">Red</lg-filter-button>
```

Comment:
This is the bit that I wasn't sure about with the `select-multiple`. I wondered if a checkbox group should still just have one form control that has an array of values as opposed to individually wired checkboxes.

I realise we don't currently have a checkbox group, so there is a bit of a gap there.

There is an example of checkbox groups in..
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/checkbox#Examples

some examples on Angular Material as well..
https://material.angular.io/components/checkbox/examples

Reply:
The filter checkbox group is built similar to the ones in the examples above. I did some changes and now you can implement the select-multiple variant either way, with each checkbox with its individual formcontrol as it is now(I left this working as it is, because I saw that in the majority of cases, users were using them like this), and with a formArray and we push the values in that array and we get the array of values like above. For the second implementation should I also update the notes on how the user should implement it on their side, as it needs a bit of logic with the pushing and removing of values in the formArray?

Reply:
It probably should be reflected in the notes, it might be worth checking with @aaronBery, @owensgit and @elenagarrone, just in case I am wrong on the array approach which I assumed would be the more commonly used one, but I may be wrong 😀

Reply:
Either way, I think it's better to have both and let the user choose. I would also choose the array one now, as it makes more sense for our needed implementation, but I guess it depends on the users scenario. 
The notes will look something like this:
![Screenshot 2020-10-12 at 12 33 42](https://user-images.githubusercontent.com/68600084/95731080-bb741f80-0c87-11eb-937e-4fe60ffdfd08.png)

Reply:
I agree with @thatguynamedandy , I think having the array would be enough but I think I'm ok leaving both the solutions.